dhall/src/Dhall/Main.hs

```diff
@@ -290,6 +296,45 @@ parseMode =
             <>  Options.Applicative.metavar "FILE"
             )
 
+    parsePath = fmap f (optional p)
+      where
+        f  Nothing    = StandardInput
+        f (Just path) = File path
+
+        p = Options.Applicative.strOption
+            (   Options.Applicative.long "path"
+            <>  Options.Applicative.help "Index all files in path recursively"
+            <>  Options.Applicative.metavar "PATH"
+            )
+
+    parseOutput = fmap f (optional p)
+      where
+        f  Nothing    = StandardOutput
+        f (Just file) = FileO file
+
+        p = Options.Applicative.strOption
+            (   Options.Applicative.long "output"
+            <>  Options.Applicative.help "Output into file. STDOUT by default"
```

Comment:
Isn't there a convention for tag file names? E.g. `fast-tags` produces a `tags` file by default.

Reply:
Add tags by default. STDOUT by default, as I thought, closer to dhall tool logic.
But I think tags by default is more useful.